doc/statistics/chatterjee_correlation.qbk

```diff
@@ -0,0 +1,74 @@
+[/
+  Copyright 2022 Matt Borland
+
+  Distributed under the Boost Software License, Version 1.0.
+  (See accompanying file LICENSE_1_0.txt or copy at
+  http://www.boost.org/LICENSE_1_0.txt).
+]
+
+[section:chatterjee_correlation Chatterjee Correlation]
+
+[heading Synopsis]
+
+``
+#include <boost/math/statistics/chatterjee_correlation.hpp>
+
+namespace boost::math::statistics {
+
+    C++17:
+    template <typename ExecutionPolicy, typename Container>
+    auto chatterjee_correlation(ExecutionPolicy&& exec, const Container& u, const Container& v);
+
+    C++11:
+    template <typename Container>
+    auto chatterjee_correlation(const Container& u, const Container& v);
+}
+``
+
+[heading Description]
+
+The classical correlation coefficients like the Pearson's correlation are useful primarily for distinguishing when one dataset depends linearly on another.
+However, Pearson's correlation coefficient has a known weakness in that when the dependent variable has an obvious functional relationship with the independent variable, the value of the correlation coefficient can take on any value.
+As Chatterjee says:
+
+> Ideally, one would like a coefficient that approaches
+its maximum value if and only if one variable looks more and more like a
+noiseless function of the other, just as Pearson correlation is close to its maximum value if and only if one variable is close to being a noiseless linear function of the other.
+
+This is the problem Chatterjee's coefficient solves.
+Let X and Y be random variables, where Y is not constant, and let (X_i, Y_i) be samples from this distribution.
+Rearrange these samples so that X_(0) < X_{(1)} < ... X_{(n-1)} and create (X_{(i)}, Y_{(i)}).
```

Comment:
@mborland : Does this render properly?  I wonder if we need to use (say) [this](https://viereck.ch/latex-to-svg/) to get the docs correct?